pokemongo_bot/tree_config_builder.py

```diff
@@ -54,6 +54,17 @@ def build(self):
                                         'See config.json.*example for more information.')
                 continue
 
+            if not task_config.get("enabled", True):
+                msg = "Task {task_name} is disabled"
+                self.bot.event_manager.emit(
+                    'task_disabled',
+                    sender=self,
+                    level='info',
+                    formatted=msg,
+                    data={'task_name': task_type}
+                )
+                continue
+
```

Comment:
@alexyaoyang is right, it will log only at startup. I've added event instead of logger.warning to color its output (yellow).